packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -563,13 +563,48 @@ export namespace EmrCreateCluster {
   }
 
   /**
-   * The launch specification for Spot instances in the instance fleet, which determines the defined duration and provisioning timeout behavior.
+   * Different allocation strategies that can be used by the EMR cluster
+   */
+  export class AllocationStrategy {
+    /** lowest-price allocation strategy. */
```

Comment:
include a link to the docs - ideally we want to provide more information to users to determine which allocation is most appropriate for their use case(s)

Reply:
Done, added link to the Cfn docs.